resources/options.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Octopeer</title>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.6/css/materialize.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" >
+
+    <script src="jquery.min.js"></script>
+    <script src="materialize.min.js"></script>
+    <script src="options.js"></script>
+</head>
+<body>
+
+<nav>
+    <div class="nav-wrapper lime">
+        <a href="" class="brand-logo center">
+            Octopeer
+        </a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+            <li class="active"><a href="options.html">Options</a></li>
+            <li><a href="">About</a></li>
+        </ul>
+    </div>
+</nav>
+
+<div class="container">
+    <p class="flow-text" style="text-align: center;">
+        Welcome to the personal settings of Octopeer. <br>
+        Please check whether all of the following settings apply to you.
+    </p>
+
+    <div class="row">
+    <div class="col s12 m6">
+        <div class="card">
+            <div class="card-content">
+                <span class="card-title">General</span> <br>
+                <input type="checkbox" id="logging" checked>
+                <label for="logging">I accept Octopeer to log data from me while reviewing pull requests on GitHub.</label>
+            </div>
+        </div>
+    </div>
+
+    <div class="col s12 m6">
+        <div class="card">
+            <div class="card-content">
+                <span class="card-title">Privacy</span> <br>
+                <p style="font-family: Roboto,sans-serif; font-size:11pt">
+                The following data can be tracked from me, while I'm reviewing pull requests on GitHub:
+                </p>
+                <ul>
+                    <li>
+                        <input type="checkbox" id="tabs" checked>
+                        <label for="tabs">other active tabs that are used.</label>
+                    </li><li>
+                        <input type="checkbox" id="comments" checked>
+                        <label for="comments"> the comments that I add to a pull request. </label>
+                    </li><li>
+                        <input type="checkbox" id="peer_comments" checked>
+                        <label for="peer_comments"> the comments of my peers that have been added to a pull request. </label>
+                    </li><li>
```

Comment:
We do not have a style check for HTML, but can I be picky and want the `</li><li>` split with a line break? :P